packages/assets-controllers/src/TokensController.ts

```diff
@@ -152,21 +119,6 @@ export class TokensController extends BaseController<
 
   private messagingSystem: TokensControllerMessenger;
 
-  private failSuggestedAsset(
-    suggestedAssetMeta: SuggestedAssetMeta,
-    error: unknown,
-  ) {
-    const failedSuggestedAssetMeta = {
-      ...suggestedAssetMeta,
-      status: SuggestedAssetStatus.failed,
-      error,
-    };
-    this.hub.emit(
-      `${suggestedAssetMeta.id}:finished`,
```

Comment:
Was this never used or have we now stopped using it?

Reply:
We have now stopped using it. This was called from within the catch statements so that we could send the error to an event handler that would reject the promise.

We can now throw directly from `watchAsset`.